### PR TITLE
Fixed Cesium rendering when clustering is turned on and the canvas is…

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?

This PR uses billboards and dynamically created SVGs to render clustering markers.   This was done to avoid rendering issues, as well as provide better looking markers.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@andrewkfiedler  @djblue 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@pklinef
@kcwire 
#### How should this be tested?

Build the Catalog UI, ingest data, perform a query, turn clustering on and switch workspaces
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

… hidden

### CODE REVIEW

File: catalog/ui/catalog-ui-search/src/main/webapp/js/widgets/cesium.mapclustering.js [hunk 1]
```diff
@@ -174,11 +170,21 @@ define(['backbone',
...
-        getEllipsoidAt40PercentOfMapView: function(width, height) {
+        getEllipsoidAt40PercentOfMapView : function(width, height) {
             return viewer.camera.pickEllipsoid(new Cesium.Cartesian2(width * 40 / 100, height / 2));
         },
-        getEllipsoidAt60PercentOfMapView: function(width, height) {
+        getEllipsoidAt60PercentOfMapView : function(width, height) {
             return viewer.camera.pickEllipsoid(new Cesium.Cartesian2(width * 60 / 100, height / 2));
+        },
+        getSvgImage : function(value) {
+            var svgString = '<svg xmlns="http://www.w3.org/2000/svg" width="25" height="25">';
```
Comment (djblue) [inline catalog/ui/catalog-ui-search/src/main/webapp/js/widgets/cesium.mapclustering.js]: We might want to use a handlebars template here. Something like: 

``` javascript
var svg = hbs.compile(template)({
  fill: value.color.toCssColorString(),
  count: value.points.length
});

return window.btoa('data:image/svg+xml;base64,' + svg);
```

would work.

Comment (bdeining) [inline catalog/ui/catalog-ui-search/src/main/webapp/js/widgets/cesium.mapclustering.js]: Added a handlebars template

**Comment by andrewkfiedler** [comment]
👍 Worked fine on my machine.

~~I ran into an issue with the way variables are being used in cesium.mapclustering though.  There are multiple maps, so each MapClustering instance should maintain certain variables separately.~~

~~Specifically these (https://github.com/codice/ddf-ui/blob/master/catalog/ui/catalog-ui-search/src/main/webapp/js/widgets/cesium.mapclustering.js#L26):~~

```
    var clusters = [];

    var entities, viewer;
```

~~Not sure if you want to tackle that with this PR.  If not, I can push up a fix.~~

Nevermind, just looked at https://github.com/codice/ddf-ui/pull/6 where it looks like that's all addressed.

**Comment by bdeining** [comment]
Rolled this into https://github.com/codice/ddf-ui/pull/6 for simplicity

**Pull request closed without merging**